dhctl/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/deckhouse/deckhouse/dhctl
 
-go 1.19
+go 1.21
```

Comment:
Maybe we will switch to 1.22? Deckhouse- controller already switch to 1.22 https://github.com/deckhouse/deckhouse/blob/main/go.mod#L3